api/src/main/java/org/openmrs/Order.java

```diff
@@ -362,6 +363,41 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated on the current date
```

Comment:
This doesn't sound exactly correct, I think this should say "indicating whether the order was activated before or on the current date"